### PR TITLE
[nnpackage] Update circle schema to 0.6

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit updates schema to support more ops and options.

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

---

Related issue: #11381

### CODE REVIEW

File: nnpackage/schema/circle_schema.fbs [hunk 1]
```diff
@@ -30,6 +30,7 @@
 // Version 0.3: SHUFFLED16x1FLOAT32 is added.
 // Version 0.4: Base up to TensorFlow Lite v2.7.0 schema.
 // Version 0.5: Base up to TensorFlow Lite v2.10.1 schema.
+// Version 0.5: Base up to TensorFlow Lite v2.13.0 schema.
```
Comment (seanshpark) [inline nnpackage/schema/circle_schema.fbs:33]:
```suggestion
// Version 0.6: Base up to TensorFlow Lite v2.13.0 schema.
```

**Comment by seanshpark** [comment]
@nnfw-bot test nncc-jammy-debug

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by glistening — APPROVED** [review]
LGTM

**Pull request merged**